test/examples/ucp_client_server.c

```diff
@@ -621,8 +701,7 @@ static int init_context(ucp_context_h *ucp_context, ucp_worker_h *ucp_worker)
     ucp_params.field_mask   = UCP_PARAM_FIELD_FEATURES     |
                               UCP_PARAM_FIELD_REQUEST_SIZE |
                               UCP_PARAM_FIELD_REQUEST_INIT;
-    ucp_params.features     = UCP_FEATURE_STREAM;
-
+    ucp_params.features     = UCP_FEATURE_STREAM | UCP_FEATURE_TAG;
```

Comment:
should not be initialized according to communication type from cmd args?

Reply:
i think it's simpler to initiate to both for simplicity